pennylane/_qubit_device.py

```diff
@@ -382,7 +382,7 @@ def estimate_probability(self, wires=None):
         # consider only the requested wires
         wires = np.hstack(wires)
 
-        samples = self._samples[:, np.array(wires)]
+        samples = self._samples[:, np.array(wires)]  # TODO: Use indices for nonconsec wires
```

Comment:
If ``samples`` is a list of 3 elements, but the wires is a sequence like [5, 10] (maybe ``5`` addressing the first qubit and ``10`` the third), then we cannot index the former by the latter. Instead, my plan is to turn the user-given ``op.wires`` into a list of indices of the wires in the device's register (like ``[0, 2]`` in this case). Then all devices should work again. 

I'm just flagging the locations where I spotted that this will be necessary. The implementation will be done in the next PR.